### PR TITLE
Add installation/troubleshooting suggestion to README

### PR DESCRIPTION
**Pull request by kytrinyx**
For people who don't do development in Python on a regular basis, their
development environment is likely woefully outdated, and dependency
management is tricky.

The shotgun approach to installation failures tends to apply.
This gives people an extra thing to try that might work.

### CODE REVIEW

**Comment by willingc** [comment]
Hi @kytrinyx,

Good suggestion 👍 Any step closer to installation success for a contributor is a good thing :smile: Thanks!

**Comment by kytrinyx** [comment]
This one worked for me (after `sudo` failed) :)

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Clone the repository, change to the directory containing the repository.
 $ pip install -r requirements.txt
```
Comment (hollenberry) [inline README.md:45]: Would it also be worth a few lines here for installing Python/Pip? 

I don't actively develop (let alone in Python), so `pip` is something I needed to go get before I was able to utilize. 

Happy to put this within its own PR, but I felt it might fit here, too.

Comment (kytrinyx) [reply README.md:45]: Maybe?

Could perhaps link to https://packaging.python.org/installing/#install-pip-setuptools-and-wheel

Comment (willingc) [reply README.md:45]: @hollenberry I would go with @kytrinyx's suggestion to link to it. To explain for multiple OS and versions, it gets wordy pretty quickly. As an FYI the Django Girls tutorial is a good resource that helps a newcomer install Python, pip, etc. :smile:

Comment (kytrinyx) [reply README.md:45]: The Django Girls one talks about (unsurprisingly) Django. I've seen the thing where people follow the entire tutorial even when they only need a small bit of it. https://tutorial.djangogirls.org/en/installation/

Comment (willingc) [reply README.md:45]: Thanks @kytrinyx for adding the link.

The section on installing Python, in particular, is very good since it is well maintained and up to date since the workshops run pretty much every weekend. https://tutorial.djangogirls.org/en/python_installation/

Comment (kytrinyx) [reply README.md:45]: Oh, right. Yes, that is excellent. I'll add that to this PR.

**Comment by kytrinyx** [comment]
I've added a link to the Django Girls' Python installation guide.

**Review by willingc — APPROVED** [review]
:smile:

**Comment by willingc** [comment]
@kytrinyx Thanks! Happy to merge :cake:
